### PR TITLE
NAS-132759 / 25.04 / Rework Installed Apps

### PR DESCRIPTION
**Pull request by bvasilenko**
**Changes:**

Used new master-detail view component on installed apps page

**Testing:**

Double check that all features of the installed Apps page work

### CODE REVIEW

File: src/app/pages/apps/components/installed-apps/installed-apps.component.html [hunk 1]
```diff
@@ -29,8 +29,11 @@
   >{{ 'Discover Apps' | translate }}</a>
 </ix-page-header>
 
-<div class="container" [ixUiSearch]="searchableElements.elements.installed">
-  <div class="table-container">
+<ix-master-detail-view
+  #masterDetailView="masterDetailViewContext"
+  [selectedItem]="selectedApp"
+>
+  <ng-container master>
```
Comment (AlexKarpov98) [inline src/app/pages/apps/components/installed-apps/installed-apps.component.html:36]: Would be good to rework that to have a separate component for `installed-apps-list`, so the `installed-apps` component html may become smaller.

As example:

```
<ix-page-header>
  <ix-all-instances-header></ix-all-instances-header>
</ix-page-header>

<ix-master-detail-view
  #masterDetailView="masterDetailViewContext"
  [selectedItem]="selectedInstance()"
>
  <ix-instance-list
    master
    [isMobileView]="masterDetailView.isMobileView()"
    [showMobileDetails]="masterDetailView.showMobileDetails()"
    (toggleShowMobileDetails)="masterDetailView.toggleShowMobileDetails($event)"
  ></ix-instance-list>

  <ng-container detail-name>
    {{ selectedInstance()?.name }}
  </ng-container>

  <ng-container detail>
    @if (selectedInstance()) {
      <ix-instance-details
        [instance]="selectedInstance()"
      ></ix-instance-details>
    }
  </ng-container>
</ix-master-detail-view>
```

**Review by AlexKarpov98 — COMMENTED** [review]
Looks good overall but can be improved.

File: src/app/pages/apps/components/installed-apps/installed-apps.component.html (hunk 1 above)
Comment (AlexKarpov98) [reply src/app/pages/apps/components/installed-apps/installed-apps.component.html:36]: As well would be good to extract bulk actions to the separate component
Example:
```
@if (checkedInstances?.length > 0) {
    <ix-instance-list-bulk-actions
      [checkedInstances]="checkedInstances"
      (resetBulkSelection)="resetSelection()"
    ></ix-instance-list-bulk-actions>
  }
```

**Review by RehanY147 — APPROVED** [review]
Seems to be working the same as before

**Pull request merged**